etc/kayobe/kolla.yml

```diff
@@ -132,6 +132,10 @@ kolla_sources:
     type: git
     location: https://github.com/stackhpc/cloudkitty-dashboard.git
     reference: stackhpc/{{ openstack_release }}
+  horizon:
+    type: git
+    location: https://github.com/stackhpc/horizon.git
+    reference: bump-angular-yoga
```

Comment:
After it's tested, let's merge the PR in Horizon and update this to stackhpc/yoga.

Reply:
Done